packages/apps-config/src/endpoints/testing.ts

```diff
@@ -222,6 +222,13 @@ export function createTesting (t: TFunction): LinkOption[] {
         Interlay: 'wss://beta.polkabtc.io/api/parachain'
       }
     },
+    {
+      info: 'polymesh',
+      text: t('rpc.polymesh', 'Polymesh ITN', { ns: 'apps-config' }),
+      providers: {
+        Interlay: 'wss://itn-rpc.polymesh.live'
```

Comment:
It should be `hosted by Polymath`

<img width="289" alt="Screenshot 2021-05-09 at 7 35 43 PM" src="https://user-images.githubusercontent.com/14581509/117575045-c1d35880-b0fd-11eb-9389-24ce3040b7be.png">

So here it should be 

```
providers: {
       Polymath: 'wss://itn-rpc.polymesh.live'
        }